energy-integration/energy-factory-mock/src/lib.rs

```diff
@@ -52,9 +52,23 @@ pub trait EnergyFactoryMock {
         self.user_energy(&user).set(&energy);
     }
 
+    #[endpoint(transferUnlockedToken)]
```

Comment:
There is no access control for this endpoint.
Is this expected behavior?

Reply:
This SC is for test purposes only, as it is a mock contract. So there is no problem. Now, we've removed the mock contracts integration in tests, so it is no longer used. Not sure if we should keep it or remove it altogether.